From ceeb03eaaa1e6bda05656ab9ec44683d2a8aa263 Mon Sep 17 00:00:00 2001
From: Kha Tran <kha.tran.px@rvc.renesas.com>
Date: Fri, 19 Jan 2018 17:38:30 +0700
Subject: [PATCH 293/557] ARM: DTS: irqchip: irqc r8a7744 IRQC device tree node

Enable a r8a7744 IRQC block by adding a device tree
node for the IRQC hardware and pins IRQ0 to IRQ9.

Signed-off-by: thongsyho <thong.ho.px@rvc.renesas.com>
Signed-off-by: Kha Tran <kha.tran.px@rvc.renesas.com>
Signed-off-by: vietn <vietn@fsoft.com.vn>
Signed-off-by: Loc Vu <loc.vu.zn@renesas.com>
---
 drivers/irqchip/irq-renesas-irqc.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/irqchip/irq-renesas-irqc.c b/drivers/irqchip/irq-renesas-irqc.c
index 52304b1..d029f1d 100644
--- a/drivers/irqchip/irq-renesas-irqc.c
+++ b/drivers/irqchip/irq-renesas-irqc.c
@@ -278,6 +278,7 @@ static int irqc_remove(struct platform_device *pdev)
 
 static const struct of_device_id irqc_dt_ids[] = {
 	{ .compatible = "renesas,irqc", },
+	{ .compatible = "renesas,irqc-r8a7744", },
 	{},
 };
 MODULE_DEVICE_TABLE(of, irqc_dt_ids);
-- 
2.7.4

